HPCC-17317 DOCS:ECLWatch Add Delete Part

g-pan opened the pull request:
Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part
New Pull request, Replacing Pull Request # 10030, 10050 & 10056 - due to
substantial changes and some technical difficulties

@JamesDeFabia  please review
@miguelvazq please review
@afishbeck please review

For convenience PDF can be reviewed at: 
http://10.240.32.243/view/Docs/job/DocDev05/CE=ubuntu-14.04-amd64-1/lastSuccessfulBuild/artifact/build-docs/Release/docs/The_ECL_Watch_Manual-6.4.0-rc1.pdf
 resulting from successful test build.

Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [x] I have added tests to cover my changes.
http://10.240.32.243/view/Docs/job/DocDev05/CE=ubuntu-14.04-amd64-1/
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part
New Pull request, Replacing Pull Request # 10030, 10050 & 10056 - due to
substantial changes and some technical difficulties

Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 7ad918bccaed293166a4ca73f0c1aa0bb0618865
Build: success
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.893 seconds |


Install hpccsystems-platform-community_6.4.0-rc3.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |94 |94 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |733 |733 |0 |
|test (thor) |627 |627 |0 |
|test (roxie) |760 |760 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

afishbeck commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -589,33 +597,299 @@ depending on where they wind up.-->
+          Active Package Map</emphasis> tab. You can validate any package map,
+          active or inactive.</para>
+
+          <para>To validate any package map:</para>
```
Not sure on exact wording, but the key is these are packagemaps that aren't yet on the system.

Something like "To validate an external packagemap"
and/or "a packagemap not yet uploaded onto the environment."

JamesDeFabia replied:
I like: "To validate a package map not yet uploaded to the environment:"

afishbeck commented:
@g-pan just one minor comment.

g-pan commented:
@afishbeck please let me know if this change is better.

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -589,33 +597,301 @@ depending on where they wind up.-->
+                <para>Select a <emphasis role="bold">Process Filter</emphasis>
+                from the drop list. The process filter determines which
+                physical Roxie clusters will actually load the
+                packagemap.</para>
```
package map

JamesDeFabia reviewed (changes requested):
Only one minor change

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: d9d7ff8fc7a6b14c60c74cbf894d9b633f51449b
Build: success
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 95.578 seconds |


Install hpccsystems-platform-community_6.4.0-rc3.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |94 |94 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |733 |733 |0 |
|test (thor) |627 |627 |0 |
|test (roxie) |760 |760 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

afishbeck reviewed (approved):
@g-pan looks good.

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 2d60b463112468b5cf97987be154a0a0c9d037fa
Build: success
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 93.462 seconds |


Install hpccsystems-platform-community_6.4.0-rc3.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |94 |94 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |733 |733 |0 |
|test (thor) |627 |627 |0 |
|test (roxie) |760 |760 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

g-pan commented:
Rebased for a nice clean merge

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 3ced03394b243cfda92562200c79e384e338a648
Build: success
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.667 seconds |


Install hpccsystems-platform-community_6.4.0-rc3.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |94 |94 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |733 |733 |0 |
|test (thor) |627 |627 |0 |
|test (roxie) |760 |760 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

miguelvazq commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -282,7 +282,7 @@ depending on where they wind up.-->
                 </itemizedlist>
 
-                <para>The Filter also also supports wild cards. </para>
+                <para>The Filter also also supports wild cards.</para>
```
You have an extra "also" in here

miguelvazq commented:
FYI ECLWAPkgParts01.jpg will not have some of the fields in them after https://github.com/hpcc-systems/HPCC-Platform/pull/10094 is merged.

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 033ad88caa080c17da718b7e63982eebefb5d1a9
Build: success
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 96.159 seconds |


Install hpccsystems-platform-community_6.4.0-rc3.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |94 |94 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |733 |733 |0 |
|test (thor) |627 |627 |0 |
|test (roxie) |760 |760 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

miguelvazq commented:
Looks fine other than my image comment.

g-pan commented:
Added https://track.hpccsystems.com/browse/HPCC-17827 for image update.
